modules/microshift-updates-troubleshooting.adoc

```diff
@@ -0,0 +1,48 @@
+//Module included in the following assemblies:
+//
+//* microshift_troubleshooting/microshift-updates-troubleshooting.adoc
+
+:_content-type: CONCEPT
+[id="microshift-troubleshooting-updates_{context}"]
+= Troubleshooting {product-title} updates
+
+In some cases, {product-title} might fail to update. In these events, it is helpful to understand failure types and how to troubleshoot them.
+
+[id="microshift-update-path-blocked-by-version-sequence"]
+== Update path is blocked by {product-title} version sequence
+{product-title} requires serial updates. Attempting to update {product-title} by skipping a minor version fails:
+
+* For example, if your current version is `4.14.5`, but you try to update from that version to `4.16.0`, the message, `upgrade from 4.14.5 to 4.16.0 is blocked` appears and {product-title} fails to start.
```

Comment:
Do we expect here an exact message that will appear in logs? Because it would be:
```
executable (4.16.0) is too recent compared to existing data (4.14.5): version difference is 2, maximum allowed difference is 1
```